### PR TITLE
[New Rule] Unusual User Calling the Metadata Service [Windows]

### PR DESCRIPTION
**Pull request by randomuserid**
## Issues

https://github.com/elastic/mechagodzilla/issues/103

## Summary

ML rule for the Windows version of the job that looks for rare username calling the metadata service

### CODE REVIEW

File: rules/ml/ml_windows_anomalous_metadata_user.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[metadata]
+creation_date = "2020/09/22"
+ecs_version = ["1.6.0"]
+maturity = "development"
```
Comment (bm11100) [inline rules/ml/ml_windows_anomalous_metadata_user.toml:4]:
```suggestion
maturity = "production"
```

File: rules/ml/ml_windows_anomalous_metadata_user.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+updated_date = "2020/09/22"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = "Looks for anomalous access to the metadata service by an unusual user. The metadata service may be targeted in order to harvest credentials or user data scripts containing secrets."
+false_positives = ["A newly installed program, or one that runs under a new or rarely used user context, could trigger this detection rule. Manual interrogation of the metadata service during debugging or troubleshooting could trigger this rule."]
```
Comment (bm11100) [inline rules/ml/ml_windows_anomalous_metadata_user.toml:11]: needs linted

**Review by bm11100 — APPROVED** [review]

File: rules/ml/ml_windows_anomalous_metadata_user.toml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/22"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = """
+Looks for anomalous access to the metadata service by an unusual user. The metadata service may be targeted in order to
+harvest credentials or user data scripts containing secrets.
```
Comment (bm11100) [inline rules/ml/ml_windows_anomalous_metadata_user.toml:12]:
```suggestion
Looks for anomalous access to the cloud platform metadata service by an unusual user. The metadata service may be targeted in order to
harvest credentials or user data scripts containing secrets.
```

Comment (bm11100) [reply rules/ml/ml_windows_anomalous_metadata_user.toml:12]: may need to re-lint after this change

**Review by devonakerr — APPROVED** [review]

**Pull request merged**